testing/basic-tenant.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# This script requires: kubectl, kind, jq
+
+#set -e
+
+yell() { echo "$0: $*" >&2; }
+
+die() {
+  yell "$*"
+  (kind delete cluster || true ) && exit 111
+}
+
+try() { "$@" || die "cannot $*"; }
+
+try kind create cluster --config kind-config.yaml
+
+echo "Installing Current Operator"
+
+# TODO: Compile the current branch and create an overlay to use that image version
+try kubectl apply -k ../resources
+
+echo "Waiting for Operator Pods to come online"
+
+try kubectl wait --namespace minio-operator \
+  --for=condition=ready pod \
+  --selector=name=minio-operator \
+  --timeout=90s
+
+echo "Installing lite tenant"
+
+try kubectl apply -k ../examples/kustomization/tenant-lite
+
+
+echo "Waiting for the tenant statefulset, this indicates the tenant is being fulfilled"
+waitdone=0
+totalwait=0
+while true; do
+  waitdone=$(kubectl -n tenant-lite get pods -o json | jq '.items | length')
+  if [ "$waitdone" -ne 0 ]; then
+    echo "Found $waitdone pods"
+    break
+  fi
+  sleep 5
+  totalwait=$((totalwait + 5))
+  if [ "$totalwait" -gt 300 ]; then
+    echo "Tenant never created statefulset after 5 minutes"
+    try false
+  fi
+done
+
+echo "Waiting for tenant pods to come online (5m timeout)"
+try kubectl wait --namespace tenant-lite \
+  --for=condition=ready pod \
+  --selector="v1.min.io/tenant=tenant-lite" \
```

Comment:
This selector is not working, by looking at the pod's labels, we should be using below selector:
`--selector="v1.min.io/tenant=storage-lite"`
Notice this time is using "storage" and not "tenant" 
I got the labels with command below:
`kubectl get pods --show-labels -n tenant-lite`
And I assume, we want to wait for these pods:

- storage-lite-ss-0-0
- storage-lite-ss-0-1
- storage-lite-ss-0-2
- storage-lite-ss-0-3

Reply:
Added new commit on top of this pull request to solve the issue